.github/workflows/docker.yml

```diff
@@ -38,3 +39,27 @@ jobs:
           push: ${{ github.event_name == 'push' }}
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+  deploy:
+    name: Development Team Deployment
+    needs: build
+    if: startsWith(github.ref, 'refs/tags/v')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Kill All Docker Containers
+        uses: appleboy/ssh-action@master
+        with:
+          host: ${{ secrets.HOST }}
+          username: ${{ secrets.USERNAME }}
+          key: ${{ secrets.DEPLOYMENTKEY }}
+          script_stop: true
+          script: |
+            containers=$(docker ps -q)
+            if [ ! -z $containers ]; then docker kill $containers; fi
+      - name: Deploy with New Image
+        uses: appleboy/ssh-action@master
+        with:
+          host: ${{ secrets.HOST }}
+          username: ${{ secrets.USERNAME }}
+          key: ${{ secrets.DEPLOYMENTKEY }}
+          script_stop: true
+          script: docker run -d --pull "always" --env-file .env ghcr.io/aquelemiguel/parrot:${{ github.ref_name }}
```

Comment:
` ${{ github.repository }}` used in the previous step can be used here right? Instead of `aquelemiguel/parrot`

Reply:
Fair suggestion, will add!